specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/groupquota.json

```diff
@@ -0,0 +1,1797 @@
+                "type": "string"
+              },
+              "Azure-AsyncOperation": {
+                "description": "URL for checking the ongoing status of the operation.\nTo get the status of the asynchronous operation, send a GET request to the URL in Azure-AsyncOperation header value.\n\nFor more info: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#asynchronous-operations",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/OperationsStatus"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "GroupQuotas"
+        ],
+        "summary": "Updates a new GroupQuotas for the name passed.",
+        "description": "Updates the GroupQuotas for the name passed. A GroupQuotas RequestId will be returned by the Service. The status can be polled periodically. The status Async polling is using standards defined at - https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#asynchronous-operations. Use the OperationsStatus URI provided in Azure-AsyncOperation header, the duration will be specified in retry-after header. Once the operation gets to terminal state - Succeeded | Failed, then the URI will change to Get URI and full details can be checked. \n Any change in the filters will be applicable to the future quota assignments, existing quota assigned to subscriptions from the GroupQuotas remains unchanged.",
+        "operationId": "GroupQuotas_Update",
+        "x-ms-examples": {
+          "GroupQuotas_Patch_Request_ForCompute": {
+            "$ref": "./examples/GroupQuotas/PatchGroupQuotas.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/MgIdInParameters"
+          },
+          {
+            "$ref": "#/parameters/GroupQuotaNameInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": " GroupQuotasPatchRequestBody",
+            "in": "body",
+            "description": "The  GroupQuotas Patch Request.",
+            "schema": {
+              "$ref": "#/definitions/GroupQuotasEntity"
```

Comment:
>GroupQuotasEntity [](http://example.com/codeflow?start=37&length=17)

Is every property of GroupQuotasEntity patch-able? If not, only add the patch-able properties to the Patch request payload.

Reply:
Yes, every property is patchable.
There are few business ruless, when patching is not allowed. An appropriate message will provided.
Example -
1. If quota was requested for GroupId/ServiceTreId - X and the Quota is approved, then this property cannot be changed, as Quota was approved fro ServiceTreeId X, it canot be changed to Y. It's like user transferring the Quota from Group X to Group Y. We can ignore the business policies and treat all the fields as changeable. 
2. Please let me know, if there any other questions.

Reply:
so we're extending the v5 common type for ProxyResource which marks id, name, type as readonly. is there any further action we need to do here?